src/components/organisms/AdminDashboard/AdminDashboard.scss

```diff
@@ -1,18 +1,37 @@
 @import "scss/constants.scss";
 
-.admin-venue {
+.AdminDashboard {
   width: 100%;
-  height: 100%;
+  display: grid;
+  height: 100vh;
+  grid-template-columns: auto;
+  padding: $spacing--lg;
+
+  position: relative;
+  overflow-y: scroll;
 
   &__header {
     display: flex;
     align-items: center;
     justify-content: space-between;
+
+    &--left {
```

Comment:
Not really a modifier, more like a `&__header-wrapper` element: doesn't change in code between being `--left`, `--right` or anything

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2278/commits/324a563bac68a0febcb9913d6c546c19bbc68b04